bc/core/utils/status/templates.py

```diff
@@ -104,3 +109,43 @@
 
 #CL{docket_id}""",
 )
+
+
+BLUESKY_FOLLOW_A_NEW_CASE = TwitterTemplate(
+    link_placeholders=[],
+    str_template="""I'm now following {docket}:
+
+{docket_link}
+
+#CL{docket_id}""",
+)
+
+BLUESKY_FOLLOW_A_NEW_CASE_W_ARTICLE = TwitterTemplate(
+    link_placeholders=["article_url"],
+    str_template="""I'm now following {docket}:
+
+Docket: {docket_link}
+
+Context: {article_url}
+
+#CL{docket_id}""",
+)
+
+BLUESKY_POST_TEMPLATE = TwitterTemplate(
+    link_placeholders=["pdf_link"],
+    str_template="""New filing: "{docket}"
+Doc #{doc_num}: {description}
+
+PDF: {pdf_link}
+
+#CL{docket_id}""",
+)
```

Comment:
If Markdown works, perhaps:

```suggestion
BLUESKY_POST_TEMPLATE = TwitterTemplate(
    link_placeholders=["pdf_link"],
    str_template="""New filing: "{docket}"
Doc #{doc_num}: {description}

[Download PDF]({pdf_link}) | [View Docket]({docket_link}]

#CL{docket_id}""",
)
```